src/modules/PortAudio/PortAudioWriter.cpp

```diff
@@ -191,7 +204,30 @@ void PortAudioWriter::pause()
 
 QString PortAudioWriter::name() const
 {
-	return PortAudioWriterName;
```

Comment:
This I don't get. Why not provide the more detailed information everywhere? I propose to add the content sample rate only on Mac (aka platforms where bitperfect audio is supported, if ever there will be more) but NOT to check for the bitperfect flag. This will tell the user at any time how sample rate is being handled during playback: conversion by CoreAudio (bitperfect OFF) or by the hardware device.

Reply:
I'll check this code at runtime :)